packages/promise/promise-tests.js

```diff
@@ -1,18 +1,24 @@
 Tinytest.addAsync("meteor-promise - sanity", function (test, done) {
   var expectedError = new Error("expected");
-  Promise.resolve("working").then(function (result) {
-    test.equal(result, "working");
-    throw expectedError;
-  }).catch(function (error) {
-    test.equal(error, expectedError);
-    if (Meteor.isServer) {
-      var Fiber = require("fibers");
-      // Make sure the Promise polyfill runs callbacks in a Fiber.
-      test.instanceOf(Fiber.current, Fiber);
-    }
-  }).then(done, function (error) {
-    test.exception(error);
-  });
+  Promise.resolve("working")
+    .then(function (result) {
+      test.equal(result, "working");
+      throw expectedError;
+    })
+    .catch(function (error) {
+      test.equal(error, expectedError);
+      if (Meteor.isServer) {
+        // TODO[FIBERS]: Remove this in 3.0
```

Comment:
As this is going to be in version 3, I don't think we need this test anymore... You could remove it already.